SU2_CFD/src/numerics/NEMO/CNEMONumerics.cpp

```diff
@@ -53,7 +53,7 @@ CNEMONumerics::CNEMONumerics(unsigned short val_nDim, unsigned short val_nVar,
     EDDY_VISC_INDEX = nSpecies+nDim+9;
 
     /*--- Read from CConfig ---*/
-    implicit   = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
+    implicit   = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
```

Comment:
This is always true for at least the first iteration.

Reply:
Yep you need the physics specific versions (_Flow, _Turb, etc.) in constructors.